pkg/azure/api/api.go

```diff
@@ -75,7 +75,7 @@ func constructAuthorizer(cloudName, userAssignedIdentityName string) (autorest.A
 
 		spToken, err := adal.NewServicePrincipalTokenFromMSIWithUserAssignedID(msiEndpoint,
```

Comment:
https://github.com/Azure/go-autorest/blob/master/autorest/adal/token.go#L693-L697 for docs on the API call.